CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 This document outlines major changes between releases.
 
+## [0.22.0] - 2022-07-25
+
+Tree service support
+
+### Fixed
+- Error logging (#450)
+- Default bucket location constraint (#463)
+- Suspended versioning status (#462)
+- CodeQL warnings (#489, #539)
+- Bearer token behaviour with non-owned buckets (#459)
+- ACL issues (#495, #553, #571, #573, #574, #580)
+- Authmate policy parsing (#558)
+
+### Added
+- Public key output in authmate issue-secret command (#482)
+- Support of conditional headers (#484)
+- Cache type cast error logging (#465)
+- `docker/*` target in Makefile (#471)
+- Pre signed requests (#529)
+- Tagging and ACL notifications (#361) 
+- AWSv4 signer package to improve compatibility with S3 clients (#528)
+- Extension mimetype detector (#289)
+- Default params documentation (#592)
+- Health metric (#600)
+- Parallel object listing (#525)
+- Tree service (see commit links from #609)
+
+### Changed
+- Cache empty bucket settings (#439)
+- Reduce number of network requests (#439, #441)
+- Reduce number of object.Head request during object upload (#)
+- Authmate renamed to s3-authmate (#518)
```

Comment:
I think the sentence should be reworded as passive voice or ‘Renamed automate to s3-authmate’?